ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.hbs

```diff
@@ -34,6 +34,17 @@
         </div>
     </div>
 </div>
+<div class="metacard-interaction interaction-copy-dms" data-help="Copies the DMS coordinates to your clipboard."
+     data-clipboard-text="{{this.dms}}">
+  <div class="interaction-icon fa fa-clipboard">
+  </div>
+  <div class="interaction-text">
+    Copy DMS Coordinates to Clipboard
+    <div>
+      <span>{{this.dms}}</span>
+    </div>
+  </div>
```

Comment:
The lines above should use 4 space indentations.

Reply:
Fixed

Reply:
I don't think that fix made it into this pull request.

Reply:
I was cherry picking commits locally after a reset and forgot that part, is it good now?